src/retraced-js.ts

```diff
@@ -56,25 +55,32 @@ export class Client {
       version: this.config.version,
     };
 
-    const response = await fetch(`${endpoint}/publisher/v1/project/${projectId}/event`, {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-        Accept: "application/json",
-        Authorization: `token=${apiKey}`,
-      },
-      body: JSON.stringify(requestBody),
-    });
+    const response = await fetch(
+      `${endpoint}/publisher/v1/project/${projectId}/event`,
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+          Accept: "application/json",
+          Authorization: `token=${apiKey}`,
+        },
+        body: JSON.stringify(requestBody),
+      }
+    );
 
     if (!response.ok) {
-      throw new Error(`Unexpected HTTP response: ${response.status} ${response.statusText}`);
+      throw new Error(
+        `Unexpected HTTP response: ${response.status} ${response.statusText}`
+      );
     }
 
-    const newEvent: NewEventRecord = await response.json() as NewEventRecord;
+    const newEvent: NewEventRecord = (await response.json()) as NewEventRecord;
```

Comment:
Parenthesis are not correct

Reply:
Why is it wrong? It should wait for json() to return before casting right?

Reply:
Yes, all good